api/commands/build.ts

```diff
@@ -55,23 +56,36 @@ export class Builder {
       this.compiledGraph.operations.map(o => this.buildOperation(o)),
       this.compiledGraph.assertions.map(a => this.buildAssertion(a))
     );
-    const allActionNames = allActions.map(n => n.name);
+
     const actionNameMap: { [name: string]: dataform.IExecutionAction } = {};
+
```

Comment:
nit: rm this new empty line? this is quite tied to the next code line.

Reply:
Done